source/dynamical_systems/include/dynamical_systems/Circular.hpp

```diff
@@ -110,7 +117,7 @@ class Circular : public DynamicalSystem<state_representation::CartesianState> {
   void set_normal_gain(double gain);
 
   /**
-   * @brief Getter of the radiuses of the limit cycle
+   * @brief Getter of the radius's of the limit cycle
```

Comment:
`radius's` is definitely more incorrect than `radiuses` - if anything, it should be `radii` 😉 but `radiuses` matches the function name at least and I don't want to break the API again by renaming the function!

Reply:
I have to talk to the guys from CLion then :smile: